cmd/promql-langserver/promql-langserver.go

```diff
@@ -44,7 +45,8 @@ func main() {
 		if err != nil {
 			log.Fatal(err)
 		}
-		handler, err := rest.CreateInstHandler(context.Background(), prometheusClient, prometheus.NewRegistry())
+		logger := kitlog.NewSyncLogger(kitlog.NewJSONLogger(os.Stderr))
```

Comment:
Maybe it would be nice to be able to choose if we want to have the log as json or not. I feel like json is not super nice to read in a consol specially concerning a log so if we are able to print in a different way it would be nice I think :)